doc/sphinx/source/FEMtheory.rst.inc

```diff
@@ -37,7 +37,7 @@ For an n-component problems in :math:`d` dimensions, :math:`f_0 \in \mathbb{R}^n
 :math:`f_1 \in \mathbb{R}^{nd}`.
 
 .. note::
-   The notation :math:`\nabla \boldsymbol v \!:\! f_1` represents contraction over both
+   The notation :math:`\nabla \bm v \!:\! f_1` represents contraction over both
    fields and spatial dimensions while a single dot represents contraction in just one,
    which should be clear from context, e.g., :math:`v \cdot f_0` contracts only over
```

Comment:
```suggestion
   which should be clear from context, e.g., :math:`\bm v \cdot f_0` contracts only over
```
Should `f` be boldened too?

Reply:
Yes, but then also in eq. (1) right above, no?

Reply:
Yeah, if you change notation, please make it consistent. I didn't read through to see everywhere else that we may use it.